tensorboard/webapp/plugins/npmi/plugin_data.proto

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+syntax = "proto3";
+
+package tensorboard;
+
+// nPMI summaries created by the `tensorboard.plugins.npmi.summary`
+// module will include `SummaryMetadata` whose `plugin_data` field has
+// as `content` a binary string that is the encoding of an
+// `NpmiPluginData` proto.
+message NpmiPluginData {
+  string title = 1;
```

Comment:
Let's include a version field as the first field just to be future-safe.

```proto
// Version `0` is the only supported version.
int32 version = 1;
```

Reply:
Done in [7b30d0b](https://github.com/tensorflow/tensorboard/pull/3799/commits/7b30d0b1bf9895c93e3821ae021794f152af1e56)